apps/comps/hooks/useSafeWagmi.ts

```diff
@@ -104,6 +103,51 @@ export function useSafeReadContract(params?: UseReadContractParameters): any {
   return useWagmiReadContract(params);
 }
 
+/**
+ * Safe wrapper for useReadContracts that handles missing WagmiProvider
+ */
+export function useSafeReadContracts(params?: any): any {
```

Comment:
Just curious why is this necessary?

Reply:
this was introduced with the cookie banner feature in https://github.com/recallnet/js-recall/pull/1431